client/js/components/procedure/StatementSegmentsList/SegmentLocationMap.vue

```diff
@@ -234,51 +234,34 @@ export default {
       })
     },
 
-    /**
-     * Restore non-updatable comments from segments relationships after update request
-     */
-    restoreComments (comments) {
-      if (comments) {
-        const segmentWithComments = {
-          ...this.segment,
-          relationships: {
-            ...this.segment.relationships,
-            comments
-          }
-        }
-        this.setItem({ ...segmentWithComments })
-      }
-    },
-
     save () {
-      this.setItem({
-        ...this.segment,
-        attributes: {
-          ...this.segment.attributes,
-          polygon: JSON.stringify(this.featuresObject)
-        }
-      })
-      const comments = this.segment.relationships.comments ? { ...this.segment.relationships.comments } : null
+      const attributes = {
+        polygon: JSON.stringify(this.featuresObject)
+      }
+      const payload = {
+        type: 'StatementSegment',
+        id: this.segmentId,
+        attributes
+      }
 
       /**
-       *  Comments need to be removed as updating them is technically not supported
-       *  After completing the request, they are added again to the store to be able to display them
+       * Vuex-json-api may include relationships in the PATCH request,
+       * and it does not support excluding them from the payload.
+       * To avoid unintentionally sending relationship data, use a dpApi method with a manually constructed payload
+       * and update the store after a successful response.
```

Comment:
:-/

Reply:
I just wanted to say that its sad, that we have to do it like this.
Thanks for the comment. its good to explain why we can't use the vuex-store here